.github/workflows/oppiabot.yml

```diff
@@ -0,0 +1,21 @@
+# Contains jobs corresponding to Oppiabot.
+
+name: Oppiabot
+
+on:
+  pull_request_target:
+    branches:
+      - develop
+
+jobs:
+  oppiabot:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: CLA Check
+        uses: oppia/oppiabot@1.2.0
+        env:
+          SHEETS_TOKEN: ${{ secrets.SHEETS_TOKEN }}
```

Comment:
Please follow up with me regarding which secrets need to be added & what their values are, otherwise this will fail when checked in.

Reply:
@gp201 Could you follow up here?
> which secrets need to be added & what their values are, otherwise this will fail when checked in.

Reply:
Secrets can be added at the organization level and the repository level. I'm not sure what approach is being taken currently. We have to consult @vojtechjelinek for the secrets.

Reply:
Since we share the CLA sheet between web and Android I think we can add this on org. level. So @seanlip should be able to add those.

Reply:
We actually have a lot of repos under the org now. It would probably better to keep these repo level to restrict access. 

@seanlip do you have any thoughts on this?

Reply:
@BenHenning Just an FYI, We have to give permission for the GitHub action to run on a repo in [this file](https://github.com/oppia/oppiabot/blob/65f1717d62f4f3358db40909e3c4228398ee14c1/constants.js#L61) in oppiabot.

Reply:
Also, shouldn’t we be passing these values into the action rather than setting them as environment variables? While the env context is limited to this script, it seems a bit more secure to pass the secrets directly to the action like we are for the github token.

Reply:
> We actually have a lot of repos under the org now. It would probably better to keep these repo level to restrict access.
> @seanlip do you have any thoughts on this?

Would it be possible to explain more why having an org-level secret results in additional access issues -- who can access it and how would they do so? Do forks and private repos have access to org-based actions but not repo-based actions?

Reply:
Hi -- I had a quick chat with @BenHenning on this and I think the core concern is about secrets for one repo in the org being leaked by another repo accidentally.

So let's go repo-level for now.

Reply:
> Also, shouldn’t we be passing these values into the action rather than setting them as environment variables? While the env context is limited to this script, it seems a bit more secure to pass the secrets directly to the action like we are for the github token.

@BenHenning 
I could add this as an issue on Github and make the change in a later PR so that it will change in the next release. I don't think it would make much of a difference, but it's always better to have it.

Reply:
@gp201 I'm fine with that. Can you please link to that issue here?

Reply:
@BenHenning The issue: https://github.com/oppia/oppiabot/issues/275